bot/bot.py

```diff
@@ -81,6 +89,22 @@ async def cache_filter_list_data(self) -> None:
         for item in full_cache:
             self.insert_item_into_filter_list_cache(item)
 
+    async def ping_services(self) -> None:
+        """A helper to make sure all the services the bot relies on are available on startup."""
+        # Connect Site/API
+        attempts = 0
+        while True:
+            try:
+                log.info(f"Attempting site connection: {attempts + 1}/{constants.URLs.connect_max_retries}")
+                await self.api_client.get("healthcheck")
+                break
+
+            except (aiohttp.ClientConnectorError, aiohttp.ServerDisconnectedError) as e:
+                attempts += 1
+                if attempts == constants.URLs.connect_max_retries:
+                    raise e
```

Comment:
Nitpicky, but it can just be
```suggestion
                    raise
```